programs/stake/src/stake_instruction.rs

```diff
@@ -341,19 +324,6 @@ declare_process_instruction!(Entrypoint, DEFAULT_COMPUTE_UNITS, |invoke_context|
                 .is_active(&feature_set::stake_redelegate_instruction::id())
             {
                 instruction_context.check_number_of_instruction_accounts(3)?;
-                if !invoke_context
-                    .feature_set
-                    .is_active(&feature_set::reduce_stake_warmup_cooldown::id())
-                {
-                    // Post feature activation, remove both the feature gate code and the config completely in the interface
```

Comment:
Same here, can you remove it from the creator and comment?

https://github.com/anza-xyz/agave/blob/e261e2704a1a874b4df5802cbeb6d119d3dc4240/sdk/program/src/stake/instruction.rs#L275-L295

... even though this will probably be removed